index.js

```diff
@@ -117,7 +117,15 @@ function sanitizeHtml(html, options, _recursing) {
   options = Object.assign({}, sanitizeHtml.defaults, options);
   options.parser = Object.assign({}, htmlParserDefaults, options.parser);
 
-  const tagAllowed = function (name) {
+  const { selfClosing } = options;
+  if (Array.isArray(selfClosing)) {
+    options.selfClosing = selfClosing.reduce((before, tagName) => ({
```

Comment:
It would be more performant to use a for loop here, avoiding the need to make a new object on every pass.

Reply:
i replaced the reduce with a for-of loop